src/shared/utils/utils.ts

```diff
@@ -121,6 +122,26 @@ export const getTemplateURL = (document: any): string | undefined => {
   }
 };
 
+export const getTemplateData = <T extends WrappedDocument<OpenAttestationDocument> | OpenAttestationDocument>(
+  document: T
+):
+  | {
+      [key: string]: any;
+    }[]
+  | {
+      [key: string]: any;
+    } => {
+  if (isWrappedV3Document(document) || isRawV3Document(document)) {
+    return document.credentialSubject;
```

Comment:
this is wrong =)

Reply:
why is this wrong? isn't wrapped and raw v3 document have the same schema and doesn't require unsalt data?